### PR TITLE
change the signed in with the icon issue#12

### PR DESCRIPTION
**Pull request by sd8917**

### CODE REVIEW

**Comment by sd8917** [comment]
@thebishaldeb Please review

File: views/partials/header.ejs [hunk 1]
```diff
@@ -6,7 +6,18 @@
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
         integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
     <link rel="stylesheet" type="text/css" href="/stylesheets/main.css">
+
+
```
Comment (thebishaldeb) [inline views/partials/header.ejs]: We have moved from Bootstrap 3 too 4. 

Please remove it.

File: views/partials/header.ejs [hunk 2]
```diff
@@ -6,7 +6,18 @@
...
+    <!-- Optional theme -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+
+
+    <!-- font awesome icon -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
```
Comment (thebishaldeb) [inline views/partials/header.ejs]: We have moved from Bootstrap 3 too 4. 

Please remove it.

File: views/partials/header.ejs [hunk 3]
```diff
@@ -49,6 +81,19 @@
...
     </div>
+     
+
+  <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <!-- Include all compiled plugins (below), or include individual files as needed -->
+    <script src="js/bootstrap.min.js"></script>
+    <!-- Latest compiled and minified JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+
```
Comment (thebishaldeb) [inline views/partials/header.ejs:86]: Same here too.

File: views/partials/header.ejs [hunk 4]
```diff
@@ -29,12 +40,33 @@
...
+                            <i class="fa fa-user-circle-o" style="color:red"></i> &nbsp; <%=currentUser.username %> &nbsp;<span class="caret"></span></a>
+                            <!-- added dropdownmenu-->
+                          <ul class="dropdown-menu">
+
+                            <li><a href="#">Post &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<span class="badge">1</span></a></li>
+                            <li role="separator" class="divider"></li>
+                            <li><a href="#">Setting</a></li>
+                            <li role="separator" class="divider"></li>
+                            <li><a href="#">Last Post &nbsp;&nbsp;<span class="badge">2 hour before</span></a></li>
+
```
Comment (thebishaldeb) [inline views/partials/header.ejs:50]: This is supposed to be dynamic and rendered from server side and thus I gave `intermediate` label. Not a static one.

I'll change the issue to `diificult` label.

File: views/partials/header.ejs [hunk 5]
```diff
@@ -49,6 +81,19 @@
...
+  <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <!-- Include all compiled plugins (below), or include individual files as needed -->
+    <script src="js/bootstrap.min.js"></script>
+    <!-- Latest compiled and minified JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+
+</body>
+</html>
+
```
Comment (thebishaldeb) [inline views/partials/header.ejs]: Closing `<body>` and `<html>` tags should not be placed here.

**Comment by thebishaldeb** [comment]
Upload a screenshot of the same too and refer to the issue in the description of the PR using # .

**Comment by sd8917** [comment]
@thebishaldeb  what change should i make ?

**Comment by thebishaldeb** [comment]
Remove bootstrap 3 and other things that I mentioned. Also the total no of posts need to be rendered from database. Check out from google how to do it using mongodb database.
